h2o-genmodel/src/main/java/hex/genmodel/CategoricalEncoding.java

```diff
@@ -0,0 +1,6 @@
+package hex.genmodel;
```

Comment:
Should this really be in a top-level packages ? Maybe somewhere in  the `algos` subpackage ? Would be hard to change in future, as this is user-facing.

Reply:
IMHO this is not an entirely bad location, it is where all the genmodel interfaces live (IGenModel, GenModel itself). It is part of the genmodel interface. It could be in algos if you think that is better.

Reply:
No need. Just wanted to point out whether there could be any issue related to the location.